chart/values.schema.json

```
@@ -1504,6 +1504,20 @@
                             "description": "How often (in seconds) to perform the probe. Minimum value is 1.",
                             "type": "integer",
                             "default": 60
+                        },
+                        "exec": {
+                            "description": "Custom exec command for livenessProbe",
```

```suggestion
                            "description": "Command for livenessProbe",
```

I'm also not sure we need to maintain the nesting of `exec.command`. I'd rather we flatten it and just have `livenessProbe.command`. My 2c, thoughts?